src/main/java/dk/cngroup/lentils/service/StatusService.java

```diff
@@ -73,4 +70,25 @@ public Boolean isStatusInDbByCypherAndTeam(final Cypher cypher, final Team team)
     public List<Status> getAllByTeam(final Team team) {
         return statusRepository.findByTeam(team);
     }
+
+    private void getStatusByCypherAndTeamAndSaveNewStatus(final Cypher cypher,
```

Comment:
Prejmenovat na `updateStatus` nebo neco podobneho. `and` by se v nazvech metod nemelo vyskytovat (Az na par vyjimek) - Single responsibility principle - rika, ze metoda, class, .. by mela delat jen jednu vec. Pokud je to vice kroku nasledujicich za sebou, pak je potreba pojmenovat cely proces.